content/source/docs/enterprise/private/reliability-availability.html.md

```diff
@@ -119,97 +119,58 @@ stores there.
-Once an upgrade has been been detected (either online or airgap), the new code
-is imported and once ready, all services on the instance are restarted running
-the new code. The expected downtime is between 30 seconds and 5 minutes,
-depending on if database updates have to be applied.
-
-Only application services are changed during the upgrade; data is not backed up
-or restored. The only data changes that may occur during are the application of
-migrations the new version might apply to the _PostgreSQL Database_.
+All data (Configuration, PostgreSQL, Blob Storage, Vault) is stored within
+Docker volumes on the instance.
 
-When an upgrade is ready to start the new code, the system waits for all terrform runs
-to finish before continuing. Once the new code has started, the queue of runs is
-continued in the same order.
+**Snapshot:** The built-in Snapshot mechanism can be used to package up
+all data and store it off the instance. The frequency of automated snapshots can
+be configured hourly such that the worst-case data loss can be as low as 1 hour.
 
-### Recovery From Failures
+**Restore:** If the instance running Terraform Enterprise is lost, the only recovery
+mechanism in demo mode is to create a new instance and use the builtin Restore
+mechanism to recreate it from a previous snapshot.
 
-#### Demo mode
+[Configure Snapshot and Restore following these instructions](./automated-recovery.html).
 
-If the instance running Terraform Enterprise is lost, the only recovery mechanism
-in demo mode is to create a new instance and use the builtin Restore mechanism to
-recreate it from a previous snapshot.
+### Mounted Disk
 
-The frequency of automated snapshots can be configured such that worst-case
-data loss can be as low as 1 hour.
+_PostgreSQL Database_ and _Blob Storage_ use mounted disks for their
+data. Backup and restore of those volumes is the responsibility of the user, the
+system does not manage that.
 
-#### Mounted Disk
+_Configuration Data_ and _Vault Data_ for the installation is stored in Docker
+volumes on the instance. The built-in Snapshot mechanism can be used to package up the
+Configuration and Vault data and store it off the instance. The built-in Restore
+mechanism can then be used to pull the configuration data back in and restore
+operations.
+[Configure Snapshot and Restore following these instructions](./automated-recovery.html).
 
 If the instance running Terraform Enterprise is lost, the presumption is that the
-volume storing the data is not lost. Because only configuration data is stored
+volume storing the data is not lost. Because only Configuration and Vault data is stored
 on the instance, we recommend using a system snapshot mechanism to provide fast
-recovery.
+recovery following these steps:
 
-The procedure here would be as follows:
+**Snapshot**
 
 - Be sure that the mounted volume being used is automatically mounted
   automatically on start.  For instance with AWS, this means having it attached
```

Comment:
Add `,` between "For instance" and "with AWS".

Reply:
I'm going to remove the `,` on this actually! You only add a comma if there are three prepositional phrases at the start of the sentence

`For example with AWS in the us-east region, you need to add a comma`